### PR TITLE
[dev-v2.9] Add rancher istio 1.19.6 dev v2.9

### PR DESCRIPTION
**Pull request by nicholasSUSE**
## Issue: 

https://github.com/rancher/rancher/issues/44138

## Manual Testing
- [x] Provision rancher on k8s (latest -1) version or lesser
- [x]  Install chart on this cluster
- [x]  Fresh install/uninstall/upgrade/downgrade of the charts
- [x]  Upgrade this cluster in rancher to k8s latest available version. Make sure the existing chart and the app is functional after cluster upgrade.
- [x] Execute P0 scenario tests

### CODE REVIEW

**Review by diogoasouza — CHANGES_REQUESTED** [review]
There's a lot of patch files missing inside the rancher-kiali-server folder, is that intentional?

**Comment by nicholasSUSE** [comment]
> There's a lot of patch files missing inside the rancher-kiali-server folder, is that intentional?

Fixed

File: packages/rancher-istio/1.19/rancher-istio/charts/Chart.yaml
Comment (lucasmlp) [inline packages/rancher-istio/1.19/rancher-istio/charts/Chart.yaml:1]: Is the chart only available with k8s clusters up to 1.27?

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

Though I would ask @diogoasouza to review it before merging. I'm not able to assess Istio functionality.

File: packages/rancher-istio/1.19/rancher-istio/charts/Chart.yaml
Comment (nicholasSUSE) [reply packages/rancher-istio/1.19/rancher-istio/charts/Chart.yaml:1]: istio Chart version 1.19.6 supports up-to k8s 1.28. 
But Rancher on the official compatibility matrix only supports up-to 1.27.

Comment (nicholasSUSE) [reply packages/rancher-istio/1.19/rancher-istio/charts/Chart.yaml:1]: @lucasmlp updated the support

**Review by diogoasouza — APPROVED** [review]

**Pull request merged**